posdealers/04-after-sales/troubleshooting-firewall.md

```diff
@@ -19,324 +19,324 @@ Host-Header für den Traffic-Manager, dieser ist DNS-Basiert und gibt HTTP-30x R
 
 ### IPs bis Juni 2020
```

Comment:
IMO we could just remove these old IPs, or at least hide them somehow with a `<details>` tag.